apps/state_mediator/test/state_mediator/integration/gtfs_test.exs

```diff
@@ -61,9 +61,9 @@ defmodule StateMediator.Integration.GtfsTest do
       assert_first_last_stop_id("CR-Haverhill", "place-north", "place-WR-0329")
       assert_first_last_stop_id("CR-Lowell", "place-north", "place-NHRML-0254")
       assert_first_last_stop_id("CR-Kingston", "place-sstat", "place-KB-0351")
-      assert_first_last_stop_id("Green-B", ["place-pktrm", "place-gover"], "place-lake")
-      assert_first_last_stop_id("Green-C", ["place-gover", "place-north"], "place-clmnl")
-      assert_first_last_stop_id("Green-D", ["place-gover", "place-north"], "place-river")
+      assert_first_last_stop_id("Green-B", ["place-gover"], "place-lake")
```

Comment:
The "first stops" here for the lines you changed no longer need to be in a list form since they've been reduced to only one entry.